phpBB/styles/all/templates/macros/icons/font.html

```diff
@@ -0,0 +1,4 @@
+{% spaceless %}
```

Comment:
think these are supposed to be in `template` not `templates`

Reply:
they should also be twig files not html files

Reply:
I mean .. https://github.com/phpbb/phpbb/pull/5545#issuecomment-552680502
> @mrgoldy just noticed that you have all the templates in the template folder. This is not right the icons folder needs moved to all/templates/macros

If you let me know where you want them I’ll adjust it. Along with the twig file extension.

Reply:
it belongs in the current folder that exsists in the software https://github.com/phpbb/phpbb/tree/master/phpBB/styles/all/template

Reply:
templates was a typo so just move the macros/iocns folders into the template folder

Reply:
We'll get there eventually :)